Support/Testing/CircleCI/install-cmake.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+cmake_version="3.7"
+cmake_package_name="cmake-${cmake_version}.0-Linux-x86_64"
+cmake_archive="${cmake_package_name}.tar.gz"
+if [ ! -e "${cmake_archive}" ] ; then
+  wget "https://cmake.org/files/v${cmake_version}/${cmake_archive}"
```

Comment:
Well that's not downloading to `/tmp`. :p

Reply:
I'd do this:

    wget --continue --output-document="/tmp/${cmake_archive}" "https://cmake.org/files/v${cmake_version}/${cmake_archive}"

Reply:
Ahh, yeah. But the dockerfile moves the script to `/tmp` and executes it there. But making it explicit is probably better, yeah.

Reply:
Yeah but just because the script is in `/tmp` doesn't mean the output of `wget` will be located there. Your working directory doesn't change even if you run a script from another directory.